CUDLR/Scripts/Console.cs

```diff
@@ -101,18 +112,35 @@ public class Console {
 
     /* Logs string to output */
     public static void Log(string str) {
+#if UNITY_WEBGL && !UNITY_EDITOR
+      Instance.m_output.Add(str);
+#else
+      LogCore(str);
+#endif
+    }
+
+    private static void LogCore(string str)
+    {
+#if !UNITY_WEBGL || UNITY_EDITOR
```

Comment:
In WebGL mode, debug logs are already shown in browser console.
So I disable log callback to avoid duplicate output.